### ISSUE TITLE
[Retail Bug Report] Profession Create All

### ISSUE THREAD

**Issue by noelheesen** [tukui-org/ElvUI#693]
### What is the issue you are having?

When trying to mass craft/produce with create all in the profession tab it stops after 1 cast. This behaviour is only happening with elvui enabled.

### What is the expected behavior?

It does not interrupt the continued casting.

### What actually happened?

It stopped the continued casting after the first cast.

### Suggested Solution/Workaround

Disable elvui if you want to do stuff with professions

### Errors

![afbeelding](https://user-images.githubusercontent.com/18031130/206809318-6be0fb5c-21f7-40f5-b86f-c1493448e865.png)

### ElvUI Changelog & Troubleshooting

- [X] I have confirmed that I have read the [Changelog](https://www.tukui.org/download.php?ui=elvui&changelog).
- [X] I have verfied ElvUI is up to date by checking /estatus.
- [X] I have verfied my issue doesn't exist in the ElvUI Issue Tracker.
- [ ] I have not read step six and I am just clicking all boxes.
- [X] I have verfied my issue doesn't exist in the default UI.
- [X] I have disabled all addons except the three called 'ElvUI', 'ElvUI Libraries', and 'ElvUI Options' by using `/edebug on`? This is important in order to rule out that any of your other addons are causing the issue. If you manage to pinpoint a conflict between ElvUI and another addon, make sure you include that information.

EDIT: provided error message

**Comment by noelheesen** [comment]
Edited with error message.

**Comment by RAtwood90** [comment]
@noelheesen 

Troubleshooting Steps (ElvUI)
Do these steps in order.

Type in chat in game /edebug on
Test - Do you still have the issue?
If you still have the issue/error - type in chat in game /estatus
Upload a full, uncropped screenshot showing the issue and the estatus.
If you are finished with debugging, you can enable your AddOns back with: /edebug off

Missing :)

**Comment by superthrust** [comment]
Not the original author, but I am having the same issue. #657 #645 are other issues referencing the same error it seems.

I have enabled Debug mode as @RAtwood90 has suggested.

Attached is my screenshot.

I can replicate the effect by opening crafting (in this case, cooking), setting a campfire, and crafting. So long as NO other ElvUI windows open, this seems to work as intended (Create All continues crafting until completed)

HOWEVER, I opened my mailbox (a ElvUI controlled window) to get more food items, and once the mailbox was closed, I attempt to "Create All" again, and it will display the error.

This happens every time. The only solution seems to be reloading the UI, then opening the crafting window AND ONLY THE CRAFTING WINDOW and clicking "Create All".

If any other ElvUI window opens during or before clicking "Create All", it seems to cause this error "

[https://i.imgur.com/exe1HuZ.jpg](image link)
